src/routes/load/components/DetailsForm/index.tsx

```diff
@@ -145,13 +145,11 @@ const DetailsForm = ({ errors, form }: DetailsFormProps): React.ReactElement =>
 }
 
 const DetailsPage = () =>
-  function LoadSafeDetails(controls: React.ReactNode, { errors, form }: StepperPageFormProps): React.ReactElement {
+  function LoadSafeDetails(controls: ReactNode, { errors, form }: StepperPageFormProps): ReactElement {
     return (
-      <>
-        <OpenPaper controls={controls}>
-          <DetailsForm errors={errors} form={form} />
-        </OpenPaper>
-      </>
+      <OpenPaper controls={controls}>
+        <DetailsForm errors={errors} form={form} />
+      </OpenPaper>
```

Comment:
Unrelated to the issue.

Reply:
Yes, just some code cleanup removing some unnecessary fragments